components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
+
+    #. Run ``composer global require symfony/var-dumper``;
+    #. Add ``auto_prepend_file = ${HOME}/.composer/vendor/autoload.php``
+       to your ``php.ini`` file;
+    #. (Don't forget to run ``composer global update`` from time to time to always
```

Comment:
With the new list layout, I would remove the parenthesis of this item. And I would reword it as follows:

```
Execute the ``composer global update`` command from time to time to
update this component and always have the latest bug fixes.
```